web/components/Feed/Feed.tsx

```diff
@@ -264,6 +278,31 @@ function Feed(props: {
     }, {});
   }
 
+  function getQueryFilters(selectedFilters: SelectedFilters<Filter>) {
```

Comment:
@katsukixyz just taking a peek at other PRs and noticed this; i slightly modified the type in my branch so it's not a generic anymore

Reply:
Hmm, right. Let me see if I can merge #108 into here and resolve the merge conflicts.